js_modules/dagit/packages/core/src/assets/LaunchAssetChoosePartitionsDialog.tsx

```diff
@@ -341,7 +358,34 @@ const LaunchAssetChoosePartitionsDialogBody: React.FC<Props> = ({
           ranges={ranges}
           setRanges={setRanges}
         />
+
+        <Box flex={{direction: 'column', gap: 32}} style={{marginTop: 32}}>
```

Comment:
32 might be too much gap, the alerts look a little too far apart.

Reply:
Yeah I agree actually, just didn't want to change the job backfill modal - will make this a bit more reasonable

Reply:
Toned this down a bit!

<img width="814" alt="image" src="https://user-images.githubusercontent.com/1037212/212176104-23861fba-de75-42b4-8e84-dbe350da7b62.png">

<img width="969" alt="image" src="https://user-images.githubusercontent.com/1037212/212176467-3fab4a28-e168-4234-a47b-e6eb5b30ce37.png">